docs/RELEASE_NOTES-v0.4.md

```diff
@@ -6,16 +6,20 @@ For all the non-critical release notes, see usual places (like github
 
 ### Lock-step upgrade requirement
 
-**Must be on version >= v0.3.3 to coordinate shutdown**
-
-Upgrading existing Federations that were created using previous (pre v0.4.x)
+Upgrading Federations that were created using previous (0.3.x)
 versions of `fedimintd` requires stopping all peers at the exact same session count
-(mint's internal consensus height), and switching to new v0.4.x release at the same time,
-before starting them again.
+(mint's internal consensus height), before simultaneously switching to new v0.4.x release
+binaries.
+
+If you are setting up a new Federation using 0.4.x version, you don't need to do anything
+special, but you do need to acknowledge being aware of this requirement (see last section
+of this document).
 
-All guardians should be available to coordinate upgrading using an out-of-band communicaitons
-channel, e.g. a group chat. Once all guardians are available, get the latest session count so
-a session in the future can be chosen to coordinate a shutdown.
+**You must first upgrade your 0.3.x Federation to version v0.3.3 or higher for functionality
```

Comment:
[nit] Existing federations can still upgrade from 0.2.1 or 0.2.2, so it might be confusing to read `0.3.x`.

Reply:
I'm going to merge and backport to releases/v0.4. If you want to update, please feel free to open another PR